### PR TITLE
Fix: Do not log password or token to the console

### LINKED ISSUE #1475: 🐛 [Bug] User-Password are logged due to apollo logging
## 🐛 Bugreport

Currently we log all user password since we log the apollo queries. We need to mask the queries in oder to prevent password exposure to the log files

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Fix: Do not log password or token to the console

### Issues

- fixes #1475 
- relates https://github.com/shellscape/apollo-log/issues/17

### Todo

- [X] None

### CODE REVIEW

File: backend/src/server/plugins.ts [hunk 1]
```diff
@@ -1,7 +1,15 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 
-import { ApolloLogPlugin } from 'apollo-log'
+import { ApolloLogPlugin, LogMutateData } from 'apollo-log'
+
+const copyInstance = (instance: any) => {
+  const data = Object.assign(
```
Comment (Mogge) [inline backend/src/server/plugins.ts]:
```suggestion
  return Object.assign(
```

File: backend/src/server/plugins.ts [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+    Object.create(Object.getPrototypeOf(instance)),
+    JSON.parse(JSON.stringify(instance)),
+  )
+  return data
```
Comment (Mogge) [inline backend/src/server/plugins.ts]:
```suggestion
```

**Review by Mogge — APPROVED** [review]
I prefer to use lodash.deepclone instead of writing code

**Comment by ulfgebhardt** [comment]
> I prefer to use lodash.deepclone instead of writing code

I agree - but we don't have that package, doe we? Do we want to include it?

**Pull request merged**